socorro/external/rabbitmq/crashstorage.py

```diff
@@ -299,3 +303,33 @@ class ReprocessingRabbitMQCrashStore(RabbitMQCrashStorage):
         False
     )
 
+
+#==============================================================================
+class ReprocessingOneRabbitMQCrashStore(ReprocessingRabbitMQCrashStore):
```

Comment:
Note that 20 lines above there's another class called `ReprocessingOneRabbitMQCrashStore` that is used by [`reprocessingjobs.py`](https://github.com/mozilla/socorro/blob/master/socorro/cron/jobs/reprocessingjobs.py#L20-L23) and done with some magic I don't understand. 
So I added a new class so I can go into the `__init__` and mess with the `config` object.